ferrilata-devilline/Models/DAOs/Level.cs

```diff
@@ -15,5 +15,45 @@ public class Level
         public Badge Badge { get; set; }
         public List<UserLevel> UserLevels { get; set; }
         public List<Pitch> Pitches { get; set; }
+
+        protected bool Equals(Level other)
```

Comment:
I'm not sure what's going on here but if you want to specify how to Level objects should be compared to each other then just override the Equals method and that's it. There you can specify when should it return true and when should it return false. There is no need to override operators.

Reply:
This was the part of the automatically generated code by Rider for overriding the Equals function - fixed